packages/core/src/events/TapEvent.ts

```diff
@@ -322,19 +314,36 @@ const applyEvent = (target: Global): void => {
     }
   }, true);
 
+  /**
+   * Listen to `keydown` event on the target
+   * Use this to fire tap event, if `enter` is clicked and prevent default if ' ' is clicked
+   */
+  target.addEventListener('keydown', (event: KeyboardEvent) => {
+    const { target, key } = event;
+
+    if (event.defaultPrevented || !(key === 'Enter' || key === ' ' || key === 'Spacebar') || !isButtonBehaviour(target)) {
```

Comment:
Maybe move this to a function, like `isButtonBehaviour()` - `isActionKey()`?

Reply:
I see you  need to check for space key later on, so we could could maybe break this out to `isEnterKey()` and `isSpaceKey()` instead. This if statement is just super long, so if we can shrink it down a bit, that would be great!

Reply:
OK